engine/src/main/scala/cromwell/webservice/CromwellApiServiceClient.scala

```diff
@@ -23,14 +25,14 @@ trait SwaggerService extends SwaggerUiResourceHttpService {
   override def swaggerUiVersion = "2.1.1"
 }
 
-object CromwellApiServiceActor {
+object CromwellApiServiceActorClient {
   def props(workflowManagerActorRef: ActorRef, config: Config): Props = {
-    Props(classOf[CromwellApiServiceActor], workflowManagerActorRef, config)
+    Props(classOf[CromwellApiServiceActorClient], workflowManagerActorRef, config)
```

Comment:
Yes please use the constructor unless there's a compelling reason not to.  If the constructor is refactored this code won't be updated automatically.